ironfish/src/networkDefinitions.ts

```diff
@@ -0,0 +1,123 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+export const TESTING = `
```

Comment:
I think one of these networks should be `dev` or `testing`, and use the genesis block in `genesisBlockDev.ts`. It can have an empty array for bootstrap nodes. That genesis block is easily mineable, so it should make it easier for developers to start up a local network of nodes on their machine.

Reply:
Added - however the genesis block in `genesisBlockDev.ts` has not been updated with the new `transactionCommitment` field in its header